src/integrations/garmr.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  bulkhead,
+  circuitBreaker,
+  ConstantBackoff,
+  handleAll,
+  IDefaultPolicyContext,
+  IPolicy,
+  Policy,
+  retry,
+  SamplingBreaker,
+  wrap,
+} from 'cockatiel';
+
+export class GarmrService {
+  instance: IPolicy;
+
+  constructor({
+    handler,
+    breakerOpts,
+    retryOpts,
+    limits,
+  }: {
```

Comment:
Due to library exposing different options for retries, circuit breakers and concurrent requests handling decided it would be best to put all options under single service.

Garmr or Garm is a wolf or dog associated and described as a blood-stained guardian of Hel (where errors go to die).